lib/nodes/null.js

```diff
@@ -18,71 +18,68 @@ var Node = require('./node')
  * @api public
  */
 
-var Null = module.exports = function Null(){};
-
-/**
- * Inherit from `Node.prototype`.
- */
-
-Null.prototype.__proto__ = Node.prototype;
-
-/**
+module.exports = class Null extends Node {
+  /**
  * Return 'Null'.
  *
  * @return {String}
  * @api public
  */
 
-Null.prototype.inspect = 
-Null.prototype.toString = function(){
-  return 'null';
-};
+  toString() {
+    return 'null';
+  };
 
-/**
- * Return false.
- *
- * @return {Boolean}
- * @api public
- */
+  get inspect() {
+    return this.toString;
```

Comment:
`return this.toString()` or `return 'null'` ?

Reply:
it cannot be `return this.toString()` since `get inspect` is a getter unless you remove the `get`.

it can also be `return "null"` for simplicity

Reply:
Sorry, my bad, missing the `get` keyword. : ) ( Thanks for fast reply

Reply:
Both `inspect` and `toString` are function in the raw code, but `inspect` is a `getter` and `toString` is a function in new code. Why ？ Did I miss anything?

Reply:
this is because class method aliases are not well supported . the best way would be to use `inspect = this.toString` but I'm not sure of the support.

Reply:
the next best thing is using getters, as I did above